Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.9-alpine
```

Comment:
Wouldn't `python3.9-slim` be a better choice than alpine linux? See: [why not to use alpine linux](https://megamorf.gitlab.io/2020/05/06/why-it-s-better-not-to-use-alpine-linux-for-python-projects)